packages/flutter_test/lib/src/controller.dart

```diff
@@ -104,6 +135,9 @@ class WidgetController {
   ///
   /// Throws a [StateError] if `finder` is empty, matches more than
   /// one state, or matches a widget that has no state.
+  ///
+  /// * Use [firstState] if you expect to match several states but only want the first.
+  /// * Use [stateList] if you expect to match several states and want all of them.
```

Comment:
You need a newline before, but not a space before. See cassowary library for instance.